requirements-ci.txt

```diff
@@ -1,10 +1,10 @@
-numpy
-scipy<1.13.0
+numpy~=2.0.0
+scipy<=1.13.0
```

Comment:
Can this also be unpinned, or is it required?

Reply:
For Scipy? I also believe it shouldn't have been pinned from the beginning according to the dependency management guidelines: https://www.notion.so/xanaduai/Dependency-Management-0756a846d19d42e490fa233afc08ca2b?pvs=4